src/BackpackServiceProvider.php

```diff
@@ -65,6 +65,14 @@ public function register()
             return new CrudPanel();
         });
 
+        $this->app->scoped('DatabaseSchema', function ($app) {
+            return new DatabaseSchemaManager();
+        });
+
+        $this->app->bind('DatabaseSchema', function ($app) {
+            return new DatabaseSchemaManager();
+        });
+
```

Comment:
Uuuu... this is interesting 👀 Why both scoped and singleton?

Reply:
Sorry, this is a left-over from the the solution I had before with both files `DatabaseSchema` and `DatabaseSchemaManager`. I will remove it.

The reason we use a facade is so that we can call non-static methods statically.